src/test/java/fr/adrienbrault/idea/symfony2plugin/tests/dic/yaml/YamlDicCompletionContributorTest.java

```diff
@@ -56,16 +56,9 @@ public void testServiceCompletion() {
 
         assertCompletionContains(YAMLFileType.YML, "services:\n" +
             "    newsletter_manager:\n" +
-            "        parent: @<caret>\n"
+            "        parent: <caret>\n"
```

Comment:
This test was incorrect. Parent key value should not start with `@` prefix:

```
  The value of the "parent" option for the "app.service.foo" service must be the id of the service without the "@" prefix (replace "@app.service.parent" with "app.service.parent").  
```

<details>
  <summary>Code snippet</summary>

  ```yaml
  services:
      app.service.parent:
          public: true
  
      app.service.foo:
          class: App\Service\FooService
          parent: '@app.service.parent'
  ```
</details>

More details: https://symfony.com/doc/current/service_container/parent_services.html